test/wdio/slot-replace-wrapper/cmp.test.tsx

```diff
@@ -1,79 +1,81 @@
-import { setupDomTests } from '../util';
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
 
 describe('slot replace wrapper', () => {
-  const { setupDom, tearDownDom } = setupDomTests(document);
-  let app: HTMLElement;
-
   beforeEach(async () => {
-    app = await setupDom('/slot-replace-wrapper/index.html');
+    render({
+      template: () => <slot-replace-wrapper-root></slot-replace-wrapper-root>,
+    });
+
+    await $('slot-replace-wrapper-root').waitForExist();
+    await browser.pause(100);
```

Comment:
I would suggest this, it's more explicit towards what we are actually waiting for. Using `pause` command is rather discouraged as it is vulnerable to race conditions.

```suggestion
    await $('.results1 a').waitForExist();
```